packages/engine/src/scene/systems/SceneObjectUpdateSystem.ts

```diff
@@ -402,15 +393,17 @@ export default async function SceneObjectUpdateSystem(world: World) {
 
   world.scenePrefabRegistry.set(ScenePrefabs.image, [
     ...defaultSpatialComponents,
-    { name: SCENE_COMPONENT_IMAGE, props: SCENE_COMPONENT_IMAGE_DEFAULT_VALUES }
+    {
+      name: SCENE_COMPONENT_IMAGE,
+      props: { source: '__$project$__/XREngine-development-test-suite/assets/sample_etc1s.ktx2' }
```

Comment:
this should be default-project, and having the __$project$__ identifier here will not properly deserialize the first time. this mutation happens on the backend.